rust/src/measurements/make_private_lazyframe/group_by/mod.rs

```diff
@@ -70,6 +70,18 @@ where
     let (middle_domain, middle_metric) = t_prior.output_space();
 
     let by = match_grouping_columns(keys.clone())?;
+
+    by.iter().try_for_each(|name| {
+        let series_domain = middle_domain.series_domain(name.clone())?;
+        let Ok(domain) = series_domain.element_domain::<CategoricalDomain>() else {
+            return Ok(())
+        };
+        if domain.encoding().is_none() {
+            return fallible!(MakeMeasurement, "Categorical encoding is data-dependent, which may reveal sensitive record ordering. Convert {} to string before grouping.", name);
```

Comment:
```suggestion
            return fallible!(MakeMeasurement, "Categorical encoding is data-dependent, which may reveal sensitive record ordering. Cast {} to string before grouping.", name);
```

Just to be consistent with the method they will need to use.